account_payment_order/models/account_move.py

```diff
@@ -207,3 +214,23 @@ def create_account_payment_line(self):
                 }
             )
         return action
+
+    def action_payment_lines(self):
+        self.ensure_one()
+        pl_ids = (
+            self.env["account.payment.line"]
+            ._search([("move_line_id", "in", self.line_ids.ids)])
+            ._result
+        )
+        return {
+            "type": "ir.actions.act_window",
+            "name": _("Payment Lines"),
```

Comment:
Instead of creating an action from scratch, requiring another translation and full definition, just read the existing action (using `_for_xml_id`).